chart/values.schema.json

```diff
@@ -0,0 +1,1019 @@
+              "title": "impersonate",
+              "type": "boolean"
+            },
+            "podRun": {
+              "default": true,
+              "description": "Required for pod playbook actions",
+              "required": [],
+              "title": "podRun",
+              "type": "boolean"
+            },
+            "readAll": {
+              "default": true,
+              "description": "for use with kubernetes resource lookups",
+              "required": [],
+              "title": "readAll",
+              "type": "boolean"
+            },
+            "secrets": {
+              "default": true,
+              "required": [],
+              "title": "secrets",
+              "type": "boolean"
+            },
+            "tokenRequest": {
+              "default": true,
+              "description": "for secret management with valueFrom",
+              "required": [],
+              "title": "tokenRequest",
+              "type": "boolean"
+            }
+          },
+          "required": [
+            "impersonate",
+            "clusterRole",
+            "clusterAdmin",
+            "tokenRequest",
+            "secrets",
+            "configmaps",
+            "readAll",
+            "podRun",
+            "exec"
+          ],
+          "title": "rbac",
+          "type": "object"
+        }
+      },
+      "required": [
+        "name",
+        "annotations",
+        "rbac"
+      ],
+      "title": "serviceAccount",
+      "type": "object"
+    },
+    "serviceMonitor": {
+      "additionalProperties": false,
+      "properties": {
+        "enabled": {
+          "default": false,
+          "required": [],
+          "title": "enabled",
+          "type": "boolean"
+        },
+        "labels": {
+          "additionalProperties": true,
+          "required": [],
+          "title": "labels"
+        }
+      },
+      "required": [
+        "enabled"
+      ],
+      "title": "serviceMonitor",
+      "type": "object"
+    },
+    "smtp": {
+      "additionalProperties": false,
+      "properties": {
+        "secretRef": {
+          "additionalProperties": false,
+          "properties": {
+            "name": {
+              "default": "incident-commander-smtp",
+              "required": [],
+              "title": "name",
+              "type": "string"
+            }
+          },
+          "required": [
+            "name"
+          ],
+          "title": "secretRef",
+          "type": "object"
+        }
+      },
+      "required": [
+        "secretRef"
+      ],
+      "title": "smtp",
+      "type": "object"
+    },
+    "upstream_push": {
+      "additionalProperties": false,
+      "description": "Configuration for pushing data to upstream\nupstream_push:\n  name: ''\n  host: ''\n  user: ''\n  password: ''\n  labels: 'key1=val1,key2=val2'",
+      "required": [],
+      "title": "upstream_push",
+      "type": "object"
+    }
+  },
+  "required": [
+    "replicas",
```

Comment:
Can we add `required: false` tags on these ?

Reply:
All the keys ? Should I just make required empty ?

But they will be based on our default values + user supplied ones, so these `required` settings would always be present

Reply:
All the keys ? Should I just make required empty ?

But they will be based on our default values + user supplied ones, so these `required` settings would always be present